pdns/auth-main.cc

```diff
@@ -291,6 +291,7 @@ static void declareArguments()
 
   ::arg().set("lua-prequery-script", "Lua script with prequery handler (DO NOT USE)") = "";
   ::arg().set("lua-dnsupdate-policy-script", "Lua script with DNS update policy handler") = "";
+  ::arg().set("lua-global-include-dir", "Path to load Lua scripts for Lua contexts") = "";
```

Comment:
I'm not sure about this text. Is it only certain file extensions? 

```suggestion
  ::arg().set("lua-global-include-dir", "Path for Lua scripts to include in Lua contexts") = "";
```